en/examples/follow_me.md

```diff
@@ -164,9 +164,9 @@ int main(int, char **)
 
     // Device got discovered.
     Device &device = dc.device();
-    std::shared_ptr<Action> action = std::make_shared<Action>(&device);
-    std::shared_ptr<FollowMe> follow_me = std::make_shared<FollowMe>(&device);
-    std::shared_ptr<Telemetry> telemetry = std::make_shared<Telemetry>(&device);
+    std::shared_ptr<Action> action = std::make_shared<Action>(device);
```

Comment:
Why not `auto action = std::make_shared<Action>(device);` ?

Reply:
No reason at all, except that this is a direct copy of your example code :-) 
https://github.com/dronecore/DroneCore/blob/develop/example/follow_me/follow_me.cpp#L53

So if the example changes, so will this.

Reply:
oh! :) Sorry. I found my mistake!